lisa/env.py

```diff
@@ -61,10 +61,10 @@ class TargetConf(MultiSrcConf, HideExekallID):
     .. code-block:: python
 
         TargetConf({{
```

Comment:
Hang on a sec, why is that a nested dict? I must have missed something because I always create my TargetConfs like so
```python
conf = TargetConf({
    "foo" : bar
})
```

Reply:
No, it's the way to have a "{" since that string is post-processed for the placeholder.

Reply:
Gotcha, is that why the indentation is now doubled?

Reply:
No, that looks like a silent rebase issue, I'll make the indentation consistent